### DISCUSSION TITLE
VUEX

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#564] [Q&A]
Buenas, se que ya se ha hablado de esto, pero no creo acabar la práctica de vuex para el jueves. 

Se que no va a afectar en nada y que yo podré terminar la práctica después y entrar al back 

Pero aún así me queda ese sentimiento por así decirlo XD. No sé si me comprendan, de que no acabe como los demás o de atraso. 

Solo vengo a comentar porque me imagino que muchos se sienten así.

Pero como ya vimos tranquilos, no hay que estresarse.

**Comment by SamaelGT** [comment]
Buenas Gloria, igualmete ando a las a contra tiempo con esta ultima practica, apenas ayer empece a darle a esta practica y tambien siento que no podre terminarla para mañana :s

**Reply by GloriaCoralCerecedo** [reply]
Si :( pero bueno poco a poco no te preocupes no creo que pase nada y aver que comentan el jueves

**Comment by JuanSalvador73** [comment]
La práctica de VUE con los perfiles de personas usando el CND me salió sin problemas, pero la de instalar VUE cli no, de hecho estoy pidiendo ayuda en otros post. Y coincido en que si es un tanto frustrante no poder terminar un proyecto por aspectos técnicos que a veces se salen de control. 

Si puedes terminar el proyecto aunque sea después del jueves, adelante. Será hasta ese día que nos digan los requisitos para pasar a la siguiente misión, sospecho que no exigirán mucho dado los tiempos apretados que tenemos y los problemas que varios hemos experimentado, pero esperemos hasta ese entonces.

Saludos :D

**Reply by GloriaCoralCerecedo** [reply]
Si así es a esperar, tampoco creo que exijan mucho, de hecho hasta cambiaron como iban a ser las cosas, para ayudarnos

**Comment by oscardanielM** [comment]
Si gustan puedo ayudarlos en donde se atoren ja no conozco mucho vue ja pero aunque sea a buscar la respuesta a su problema podria apoyarlos

**Reply by GloriaCoralCerecedo** [reply]
Muchas gracias, de verdad jaja

**Reply by JuanSalvador73** [reply]
Explorer, te tomo la palabra.

Tengo este problema - lo comenté en otros post - No me reconoce VUE al querer crear un proyecto:

![error_vue](https://user-images.githubusercontent.com/99113565/160979449-f3ae374b-f3cf-4ca9-8671-754df18c77b5.png)

¿Que hice?

1. Bajar e instalar Node en Windows 10 en disco D, no C, no creo ese sea el problema.
2. Instalar VUE usando CMD - línea de comandos -
3. Abrir Visual Studio Code, y crear proyecto, - es cuando me marca ese error de arriba -

Ya llevo días buscando la respuesta en Google, ya he comentado en otros post, las soluciones van desde borrar e instalar de nuevo Node, VUE, hasta irse a modificar las variables en el path, y nada.

No se si hice los pasos correctos según el MC o me salté algo o el orden es incorrecto.

Saludos.

**Reply by oscardanielM** [reply]
Perfecto je no uses el powershell, a mi me paso lo mismo hasta que corri linea de comandos con permisos de administrador o en su defecto desde visual estudio code el comanprod

**Reply by JuanSalvador73** [reply]
¿Entonces todo lo corro desde la terminal de Visual Studio Code? Voy a intentarlo, gracias, avisaré aquí si tuve suerte :D

**Reply by oscardanielM** [reply]
Si mira te muestro 
![image](https://user-images.githubusercontent.com/36013303/160981925-164761d8-3a69-4063-b641-a4b4295fb59a.png)
Replique ese error 

En visual studio code cuando abres una terminal te abre la de powershell pero sale otra parte donde trae el signo de mas ahi pudes darle para abrir una con Command Prompt y desde ahi si reconoce el comando de vue create

![image](https://user-images.githubusercontent.com/36013303/160982200-7d8869e0-174b-468e-acad-24e80de56985.png)

**Reply by JuanSalvador73** [reply]
Lo hice, me marca un error anterior:

![error_vue_2](https://user-images.githubusercontent.com/99113565/160985763-3fc543ce-bb0e-4c96-9df1-97e5e75acc14.png)

Voy a ver si desinstalo todo y empiezo de nuevo. Pero la parte de cambio de terminal si parece ser parte del problema.

**Reply by oscardanielM** [reply]
ok, al parecer si tienes problemas con el npm, si seria mejor que instales todo eso otravez con sus versiones actuales

**Comment by JonathanGrimaldo** [comment]
Hola 👋🏻 de mi lado tampoco concluí la practica, por tiempos en mi trabajo 😮‍💨  pero de igual forma si no alcanzamos a concluirla en la fecha establecida, sería bueno crear el repo 🤓  y continuar con la practica, nos beneficia en el lado de aprender y terminar la practica en lo que se habilita la próxima 😄

**Comment by carlos-gmx3** [comment]
Yo de plano me compré un curso porque baby steps
La parte de los estados quedo super bien entendida, variables globales que pueden ser accedidas desde cualquier componente pero...
y lo demás? temas como v-model v-bind para formularios eran un misterio para mi, tenía la idea de combinarlos con un v-for y es posible pero necesitaba ciertos conocimientos para poder aplicarlo en un componente... y  si bien hay cursos de vue en youtube muchos son de Vue 2, traté de seguirlos pero hay algunas diferencias, ciertos ejercicios si los puedes hacer pero otros no debido a los ajuste que hicieron así que aplique tarjetazo con el curso xD
![imagen](https://user-images.githubusercontent.com/99064239/161454674-573a4025-76e4-48a9-b922-6d29b39892b7.png)
